op-test

```diff
@@ -224,6 +226,22 @@ class DefaultSuite():
     def __init__(self):
         self.s = unittest.TestSuite()
         self.s.addTest(SkirootSuite().suite())
+
+        conf = OpTestConfiguration.conf
+        target = conf.args.os_repo if conf.args.os_repo else conf.args.os_cdrom
+        if conf.args.host_scratch_disk and target:
+            if "ubuntu" in target.lower():
+                self.s.addTest(InstallUbuntu.InstallUbuntu())
+                print 'InstallUbuntu added to default suite'
+            elif "rhel" in target.lower():
+                self.s.addTest(InstallRhel.InstallRhel())
+                print 'InstallRhel added to default suite'
+            elif "hostos" in target.lower():
+                self.s.addTest(InstallHostOS.InstallHostOS())
+                print 'InstallHostOS added to default suite'
+            else:
+                print 'Could not determine os name, skipping OS install..'
+
```

Comment:
Can we have this in a function, so that can be used by multiple test suites if --scratch-disk specified.
Currently it is in install host & run default suite and also it can be like install host & run just HostSuite(By excluding skiroot suite),. I think we just want InstallHost suite like FlashFirmware and we can use it else where.

Reply:
Good idea, I'll add this in.